Sources/BraintreeCard/BTCardClient.m

```diff
@@ -237,4 +233,24 @@ - (BOOL)isGraphQLEnabledForCardTokenization:(BTConfiguration *)configuration {
     return graphQLFeatures && [graphQLFeatures containsObject:BTCardClientGraphQLTokenizeFeature];
 }
 
+- (NSError *)cardTokenizationErrorForError:(NSError *)error {
```

Comment:
I find this name a little confusing. Can you explain the `ErrorForError` bit?

Reply:
We updated this helper method name to clarify. Let us know if it needs more refining 👍